consensus/ConsensusTableUpdater.py

```diff
@@ -0,0 +1,338 @@
+import progressbar, sys
+from molgenis import client as molgenis
+from yaspin import yaspin
+from termcolor import colored, cprint
+from consensus.DataRetriever import DataRetriever
+from consensus.ConsensusTableGenerator import ConsensusTableGenerator
+from consensus.MolgenisConfigParser import MolgenisConfigParser as ConfigParser
+from consensus.HistorySorter import HistorySorter
+from consensus.ConsensusReporter import ConsensusReporter
+
+
+class ConsensusTableUpdater:
+    """The ConsensusTableUpdater uploads consensus and consensus comments by first creating a csv file for them"""
```

Comment:
This class does a bit much. I think there's a couple of distinct responsibilities:
* Transform input to data frames or lists of dicts (much easier to test as well)
* Write to CSV
* Upload CSV